test/system/users_test.rb

```diff
@@ -0,0 +1,9 @@
+require "application_system_test_case"
+
+class UsersTest < ApplicationSystemTestCase
```

Comment:
_nitpick_ _suggestion_

Please remove this file.

- Currently the feature for logged in users is not yet used or promoted in the website
- In general we should not have a commented test as in case of a refactoring this will just add an extra action without any benefit for the developer